pages/general/index.mdx

```diff
@@ -1,9 +1,24 @@
 import IncompletePage from '@components/IncompletePage.tsx'
+import IncompleteSection from '@components/IncompleteSection.tsx'
+import Link from "next/link";
 
 # Willkommen
+Herzlich willkommen beim BWINF-Guide, einem Leitfaden, der dir mit Tutorials und ähnlichen Ressourcen während deiner Teilnahme am BWINF zur Seite steht. Dieser Guide wurde von ehemaligen Teilnehmern ins Leben gerufen und befindet sich derzeit noch in der Entwicklungsphase.
 
-Herzlich Willkommen auf dem BWINF Guide, ein Guide, welcher euch bei eurer Teilnahme am BWINF mit Tutorials und Ähnlichem unterstützen sollte. Dieser Guide wurde von ehemaligen Teilnehmern erstellt und ist derzeit noch in seinen Anfangsstadien. Falls euch ein Fehler in dem Guide auffallen sollte, oder ihr eine Idee für ein Thema habt, worüber ein Artikel geschrieben werden sollte, könnt ihr gerne ein Pull-Request auf Github erstellen.
+## Der Bundeswettbewerb Informatik
+<IncompleteSection />
```

Comment:
sollen wir wirklich den Wettbewerb noch erklären?

Reply:
Denke mal das gehört einfach dazu. Obwohl das für viele vielleicht überflüssig ist, gibt es vielleicht doch welche die auf unserer Seite landen und noch nicht viel Ahnung vom BWINF haben. Da ist eine kurze Einführung nicht schlecht. USACO Guide macht das btw auch https://usaco.guide/general/usaco-faq#q-what-is-the-usaco

Reply:
ok ich bin dafür dass es in eine eigene Datei kommt

Reply:
Klingt gut